app/Containers/AppSection/Authorization/UI/API/Routes/AttachPermissionsToUser.v1.private.php

```diff
@@ -22,6 +22,6 @@
 use App\Containers\AppSection\Authorization\UI\API\Controllers\AttachPermissionsToUserController;
 use Illuminate\Support\Facades\Route;
 
-Route::post('users/permissions/attach', [AttachPermissionsToUserController::class, 'attachPermissionsToUser'])
+Route::post('users/{id}/permissions/attach', [AttachPermissionsToUserController::class, 'attachPermissionsToUser'])
```

Comment:
The intent is reflected in the http verb, I think we can remove the attach in the url here.

Reply:
Maybe PATCH would be even better here?